### PR TITLE
update: public safety html

### PR DESCRIPTION
**Pull request by eneemann**
Add sections for Fire Response Areas and EMS Response Areas

### CODE REVIEW

**Review by steveoh — COMMENTED** [review]
don't forget about the [index page](https://deploy-preview-2071--gis-utah-gov.netlify.app/data/society/). Nevermind i see it's in the public safety area.

File: data/society/public-safety/index.html [hunk 1]
```diff
@@ -121,7 +133,51 @@ <h3 id="FireStations">Fire Stations</h3>
...
+  %}
+</div>
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="FireResponseAreas">Fire Response Areas</h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <img class="inline-text-left outline" src="{% link images/fire_areas_90.png %}" alt="Fire Response Areas"
+      loading="lazy" />
+    <p>Fire Response Areas in Utah represent the primary agency, typical a local fire department, that will respond to
```
Comment (steveoh) [inline data/society/public-safety/index.html:146]:
```suggestion
    <p>Fire Response Areas in Utah represent the primary agency, typically a local fire department, that will respond to
```

File: data/society/public-safety/index.html [hunk 2]
```diff
@@ -121,7 +133,51 @@ <h3 id="FireStations">Fire Stations</h3>
...
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <img class="inline-text-left outline" src="{% link images/ems_areas_90.png %}" alt="EMS Response Areas"
+      loading="lazy" />
+    <p>EMS Response Areas in Utah represent the primary agency that will respond to medical emergencies at a given location.
+      The dataset was created to provide response recommendations for the Next-Generation 911 system and was compiled from a
+      variety of data sources. The response areas represent a variety of service levels (paramedic, advanced EMT, with or without
+      transport capability, etc.), but generally include 911 scene response. The data was compiled using written descriptions
+      from the licensing database of the Bureau of Emergency Medical Services and Preparedness <a href="https://bemsp.utah.gov/">(BEMSP)</a>,
+      which is part of the Utah Department of Health and Human Services' (DHHS). In some instances, the BEMSP data was incomplete
+      or the written descriptions were inaccurate, contradictory, ambiguous, or failed to close.  To make the data more complete,
```
Comment (steveoh) [inline data/society/public-safety/index.html:173]:
```suggestion
      or the written descriptions were inaccurate, contradictory, ambiguous, or failed to close. To make the data more complete,
```

**Review by steveoh — COMMENTED** [review]
👌🏼

**Comment by eneemann** [comment]
This PR and #2073 should be ready to go.  I've updated the AGOL IDs and the links/dowloads seem to be working as expected.

**Review by steveoh — APPROVED** [review]

**Pull request merged**